src/generic/upsell-bullets/UpsellBullets.jsx

```diff
@@ -0,0 +1,107 @@
+  );
+}
+
+// Must be child of a <ul className="fa-ul">
+export function VerifiedCertBullet() {
+  const verifiedCertLink = (
+    <a className="inline-link-underline font-weight-bold" rel="noopener noreferrer" target="_blank" href={`${getConfig().MARKETING_SITE_BASE_URL}/verified-certificate`}>
+      <FormattedMessage
+        id="learning.generic.upsell.verifiedCertBullet.verifiedCert"
+        defaultMessage="verified certificate"
+        description="Bolded words 'verified certificate', which is the name of credential the learner receives."
+      />
+    </a>
+  );
+  return (
+    <li className="upsell-bullet">
+      <CheckmarkBullet />
+      <FormattedMessage
+        id="learning.generic.upsell.verifiedCertBullet"
+        defaultMessage="Earn a {verifiedCertLink} of completion to showcase on your resumé"
+        description="Bullet showcasing benefit of earned credential."
+        values={{ verifiedCertLink }}
+      />
+    </li>
+  );
+}
+
+// Must be child of a <ul className="fa-ul">
+export function UnlockGradedBullet() {
+  const gradedAssignmentsInBoldText = (
+    <span className="font-weight-bold">
+      <FormattedMessage
+        id="learning.generic.upsell.unlockGradedBullet.gradedAssignments"
+        defaultMessage="graded assignments"
+        description="Bolded words 'graded assignment', which is the name of unlocked feature with upgrade."
+      />
+    </span>
+  );
+  return (
+    <li className="upsell-bullet">
+      <CheckmarkBullet />
+      <FormattedMessage
+        id="learning.generic.upsell.unlockGradedBullet"
+        defaultMessage="Unlock your access to all course activities, including {gradedAssignmentsInBoldText}"
+        description="Bullet showcasing benefit of additional course material."
+        values={{ gradedAssignmentsInBoldText }}
+      />
+    </li>
+  );
+}
+
+// Must be child of a <ul className="fa-ul">
+export function FullAccessBullet() {
+  const fullAccessInBoldText = (
+    <span className="font-weight-bold">
+      <FormattedMessage
+        id="learning.generic.upsell.fullAccessBullet.fullAccess"
+        defaultMessage="Full access"
+        description="Bolded phrase 'Full access'."
+      />
+    </span>
+  );
+  return (
+    <li className="upsell-bullet">
+      <CheckmarkBullet />
+      <FormattedMessage
+        id="learning.generic.upsell.fullAccessBullet"
+        defaultMessage="{fullAccessInBoldText} to course content and materials, even after the course ends"
+        description="Bullet showcasing upgrade lifts access durations."
+        values={{ fullAccessInBoldText }}
+      />
+    </li>
+  );
+}
+
+// Must be child of a <ul className="fa-ul">
+export function SupportMissionBullet() {
+  const missionInBoldText = (
+    <span className="font-weight-bold">
+      <FormattedMessage
+        id="learning.generic.upsell.supportMissionBullet.mission"
+        defaultMessage="mission"
+        description="Bolded word 'mission'."
```

Comment:
A little more about the context of the word "mission" would be helpful here (to say that it supports our goal rather than some other meaning, such as buildings that house or were built by missionaries)

Reply:
Good point. Apparently, TIL it can also mean an embassy too. We wouldn't want to cause a diplomatic incident.

```suggestion
        description="Bolded word 'mission', which is the bolded portion of a bullet point encouraging the learner to support the goals of the website."
```